src/app/_components/FeaturedEcosystemProjects.tsx

```diff
@@ -1,34 +1,44 @@
-import { Badge } from '@/components/Badge'
-import { Heading } from '@/components/Heading'
-import { TextLink } from '@/components/TextLink'
+import { MagnifyingGlass } from '@phosphor-icons/react/dist/ssr'
+
+import { Card } from '@/components/Card'
+import { CardLayout } from '@/components/CardLayout'
 
 import { getEcosystemProjectsData } from '@/utils/getEcosystemProjectData'
+import { shuffleArray } from '@/utils/shuffleArray'
 
 import { PATHS } from '@/constants/paths'
 
 const ecosystemProjects = getEcosystemProjectsData()
-
-const featuredEcosystemProjects = ecosystemProjects.filter(
-  (project) => project.featured,
-)
+const FEATURED_PROJETS_LIMIT = 6
 
 export function FeaturedEcosystemProjects() {
-  if (featuredEcosystemProjects.length === 0) {
+  const featuredEcosystemProjects = shuffleArray(ecosystemProjects).slice(
+    0,
+    FEATURED_PROJETS_LIMIT,
+  )
+
+  if (featuredEcosystemProjects.length < FEATURED_PROJETS_LIMIT) {
```

Comment:
If there are 5 `featuredEcosytemProjects` and the` FEATURED_PROJECTS_LIMIT` is 6, we show No "featured projects available"?

Reply:
Yes. This will never happen but I understand the confusion. I will revert back to `=== 0`